src/db.rs

```diff
@@ -49,11 +49,16 @@ pub struct KbsDb {
 
 impl KbsDb {
     pub async fn new() -> Result<Self> {
-        let db_type = env::var("KBS_DB_TYPE").expect("KBS_DB_TYPE not set");
-        let host_name = env::var("KBS_DB_HOST").expect("KBS_DB_HOST not set");
-        let user_name = env::var("KBS_DB_USER").expect("KBS_DB_USER not set.");
-        let db_pw = env::var("KBS_DB_PW").expect("KBS_DB_PW not set.");
-        let data_base = env::var("KBS_DB").expect("KBS_DB not set");
+        let db_type = env::var("KBS_DB_TYPE")?;
+        let host_name = env::var("KBS_DB_HOST")?;
+        let user_name = env::var("KBS_DB_USER")?;
+        let db_pw = env::var("KBS_DB_PW")?;
+        let data_base = env::var("KBS_DB")?;
+        let max_conns_str = match env::var("KBS_DB_MAX_CONNS") {
+            Ok(mc) => mc,
+            Err(_e) => "1000".to_string(),
```

Comment:
It's slightly odd that we're passing this as a string and then turning it into a u32 right below. Maybe just have this match define `max_conns` and then put the conversion below into the `Ok` block and just put 1000 as `u32` into the `Err` block?

Reply:
I condensed this to a single match statement.